src/components/Article.jsx

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
 
 export default class Article extends React.Component {
+
+  // dangerouslySetInnerHTML() introduces the possibility for a
+  // cross-site scripting (XSS) attack if improperly used.
+  // By using a createMarkup() we ensure the return of sanitized
+  // data before rendering.
+  createMarkup(text) {
+    return {__html: text};
```

Comment:
The function creates a wrapper object around the data with a single key `__html:` so that `dangerouslySetInnerHTML` cannot be passed something like `getUserName()` directly. It's taken from the [React docs](https://facebook.github.io/react/tips/dangerously-set-inner-html.html).

Looking at it now though, is there a way to ensure that the text passed into the `createMarkup()` function is a string in JavaSript/React? Maybe it would be better to write the function as
```
createMarkup() {
    return {__html: this.props.html};
}
```
instead?